module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaServiceAppRoleAssignedTo.md

```diff
@@ -9,7 +20,19 @@ schema: 2.0.0
 # Get-EntraBetaServiceAppRoleAssignedTo
 
 ## Synopsis
-{{Fill in the Synopsis}}
+
+Gets app role assignments for this app or service, granted to users, groups and other service principals.
+
+For delegated scenarios, the calling user needs at least one of the following Microsoft Entra roles.
+
+- Directory Synchronization Accounts
+- Directory Writer
+- Hybrid Identity Administrator
+- Identity Governance Administrator
+- Privileged Role Administrator
+- User Administrator
+- Application Administrator
+- Cloud Application Administrator
 
```

Comment:
```suggestion
```

Reply:
The `synopsis` section should be short and succinct. Otherwise, the command summary description would be too long like this summary of commands.

![image](https://github.com/user-attachments/assets/ee7f3001-8b02-472a-af49-553d3edcc7b7)

Reply:
changes done.